apps/emqx_durable_storage/src/emqx_ds_replication_layer.erl

```diff
@@ -47,8 +47,8 @@
 %% records over the wire.
 
 %% tags:
--define(stream, stream).
--define(it, it).
+-define(stream, 1).
+-define(it, 2).
```

Comment:
Tiny nit. What do you think making them UPPERCASE to disambiguate them from keys?

Reply:
:heavy_check_mark: